app/posts/manage-teacher-training-applications/2021-02-25-filter-by-subject.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Filtering applications by subject
+description: Let users filter the applications list by the course’s subject
+date: 2021-02-25
+---
+
+Users need to be able to filter their applications by subject.
+
+## Course data analysis
+
+Providers using the Manage service offer approximately 40 subjects at primary and secondary levels.
+
+The number of subjects offered varies between providers.
+
+- Average number of subjects offered: 10
+- Maximum number of subjects offered: 28
+- Minimum number of subjects offered: 1
+
+Some courses are made up of multiple subjects, for example:
+
+- biology and physical education
+- business studies and economics
+- English and drama
+- maths and physics
+
+Some courses are made up of multiple subjects and are also labelled differently, for example:
+
+- biology, chemistry and physics together is commonly labelled ‘combined science’
+- French, German and Spanish together is commonly labelled ‘modern languages’
+
+Renaming subjects can be problematic. For example, ‘primary’ (code 00) is often renamed to ‘primary (5-11)’ or ‘primary (7-11)’. There are also instances where ‘primary’ is renamed to a subject that already exists, such as ‘primary with mathematics’ (which is also code 03).
```

Comment:
A bit more explanation would be useful here. I don't think it's clear how subjects are renamed or why providers do it. Could we link to more information and possibly a list of subjects and codes?

Reply:
Overall this opening section is interesting, but it would be good to understand how it feeds into the design. Are we showing the 'archetypal' subject names, or the ones created by providers? How are we handling the ones made of multiple subjects and the unusual ones like 'modern languages'?

Reply:
This has now been addressed.